app/controllers/v3/apps_controller.rb

```diff
@@ -225,6 +225,19 @@ def restart
     raise CloudController::Errors::ApiError.new_from_details('RunnerUnavailable', 'Unable to communicate with Diego')
   end
 
+  def clear_buildpack_cache
+    app, space = AppFetcher.new.fetch(hashed_params[:guid])
+    app_not_found! unless app && permission_queryer.can_read_from_space?(space.id, space.organization_id)
+    unauthorized! unless permission_queryer.can_delete_buildpack_cache?(space.id)
+    suspended! unless permission_queryer.is_space_active?(space.id)
+
+    delete_job = Jobs::V3::BuildpackCacheDelete.new(app.guid)
+    Jobs::Enqueuer.new(delete_job, queue: Jobs::Queues.generic).enqueue
+    VCAP::AppLogEmitter.emit(app.guid, "Enqueued job to delete app buildpack cache with guid #{app.guid}")
```

Comment:
I would rephrase this to make clear that the guid is the app guid and not the guid of the buildpack cache.
Maybe `Enqueued job to delete buildpack cache of app with app guid #{app.guid}`.